signup/signupController.js

```diff
@@ -23,10 +70,29 @@ define('signupController', [], function() {
 				}
 			});
 
-			//TODO assume english for right now until support for other languages is added
+			// The country question has to give the treatyType instead of the entire country object
+			const countryQuestionDatabaseId = "6";
+			const indexOfCountryQuestionInQuestionsArray = questions.findIndex((question) => question.id === countryQuestionDatabaseId);
+			if (questions[indexOfCountryQuestionInQuestionsArray]) {
+				let country = questions[indexOfCountryQuestionInQuestionsArray].value;
+
+				// These ids were pulled manually from the database
+				let answerDatabaseId = -1;
+				if(country.treatyType === 'china') answerDatabaseId = "11";
+				else if(country.treatyType === 'india') answerDatabaseId = "12";
+				else if(country.treatyType === 'treaty') answerDatabaseId = "13";
+				else if(country.treatyType === 'non-treaty') answerDatabaseId = "14";
```

Comment:
This code here is pretty fragile... We do use the IDs in the HTML to store the question/possible answer IDs. But is there a way to clean this up, maybe and make it less bad?

Reply:
The way that this `questions` object is set up is a key-value where the key is the DB questionId and the value is the DB answerId--I figured I'd keep with that rather than making a random exception for this and doing stuff on the backend